src/IniFileParser.Tests/Unit/Model/SectionDataTests.cs

```diff
@@ -116,7 +115,7 @@ public void try_removing_non_existing_key_from_section()
             Assert.That(sd.Keys.ContainsKey("asdf"), Is.False);
         }
 
-        [Test]
+        [Test, Ignore("The default behaviour of INI should be empty string, not null")]
```

Comment:
See below, but you should unignore this as we need to revert to returning a null value

Also, next time, instead of ignoring the test will be nice to change it to test the new behaviour 😄